packages/components/src/layouts/Navigation/Tab/TabBar/DesktopTabItem.tsx

```
@@ -36,11 +42,15 @@ export function DesktopTabItem(
     actionList,
     avatarSrc,
     showAvatar = false,
+    onPress,
+    onClose,
     ...rest
   } = props;
 
   const stackRef = useRef<TamaguiElement>(null);
   const openActionList = useRef<() => void | undefined>();
```

Biome

[error] void is confusing inside a union type.

Unsafe fix: Use undefined instead.


(lint/suspicious/noConfusingVoidType)